manage_rancher.py

```diff
@@ -285,7 +287,12 @@ def start_new(session: str, userid: str, prespawn: Optional[bool] = False):
     labels["traefik.http.routers." + userid + ".entrypoints"] = u"web"
     container_config['launchConfig']['labels'] = labels
     container_config['launchConfig']['name'] = name
-    container_config['launchConfig']['imageUuid'] = "docker:{}".format(cfg['image'])
+    if (cfg['image_tag'] is not None and cfg['image_tag'].strip() != ''):
```

Comment:
I would move this .strip() into the merge_env_cfg() function and after the type based assignments, perform the .strip() on the value there. It does the cleaning of the input right at the start and you can just test for an empty string any time afterwards.

Reply:
Could we just strip() all the string environment variables?  See my latest change.

Reply:
I thought of the same thing, and was a little wary of changing behavior beyond what you wanted to work on. I think its a reasonable idea, but I didn't want to stick you with some firefighting in case there's some current behavior that is tied to whitespace. There *shouldn't* be, but I was being paranoid with your your.

Reply:
I'm willing to do the firefighting if there's some weird reliance on whitespace (which I agree is very unlikely).